src/cache/redisConfig.ts

```diff
@@ -0,0 +1,40 @@
+import { createClient } from 'redis';
+import { redisHost, redisPort, redisPassword } from '../config';
```

Comment:
```suggestion

import { redisHost, redisPort, redisPassword } from '../config';
```
It's good practice to separate package imports from locals